devops/scripts/shellcheck.sh

```diff
@@ -2,18 +2,26 @@
 
 set -e
 
+USE_PODMAN="${USE_PODMAN:-}"
+
+# Allow opting into using podman with USE_PODMAN=1
+if  [[ -n "${USE_PODMAN}" ]]; then
+    DOCKER_BIN="podman"
+else
+    DOCKER_BIN="docker"
+fi
+export DOCKER_BIN
```

Comment:
Nit:  unnecessary `export`?  `$DOCKER_BIN` is only otherwise used in [`dev-shell` since #6216](https://github.com/freedomofpress/securedrop/pull/6216/files#diff-520aecbdbfb8d38a696ef31c20741145bdb40e99240ca7d5f7ddfbc207b79e9b), where it's defined based on the same `$USE_PODMAN` as here.  :-)  And your changes here remove the `bash -c` subprocessing.  All three modes work without this `export`.

Reply:
I think my first version of this patch just added podman support without the other refactoring, so it needed export since `run_native_or_in_docker` was being exported too. But now it's no longer necessary since we're not exporting that either! Thanks for catching.